tests/ui/abi/unsupported.rs

```diff
@@ -8,6 +8,8 @@
 // [aarch64] compile-flags: --target=aarch64-unknown-linux-gnu --crate-type=rlib
 // [arm] needs-llvm-components: arm
 // [arm] compile-flags: --target=armv7-unknown-linux-gnueabihf --crate-type=rlib
+
+#![allow(internal_features)]
```

Comment:
oof. How about just allowing these on all tests like we do with the `dead_code` lints?